src/main/java/dev/jbang/cli/Export.java

```diff
@@ -660,6 +660,16 @@ void renderBuildFile(BuildContext ctx, Path projectDir, String fullClassName) th
 		boolean isGroovy = ctx.getProject().getMainSource() instanceof GroovySource;
 		boolean isKotlin = ctx.getProject().getMainSource() instanceof KotlinSource;
 		String kotlinVersion = isKotlin ? ((KotlinSource) ctx.getProject().getMainSource()).getKotlinVersion() : "";
+		StringBuilder jvmArgs = new StringBuilder();
+		if (ctx.getProject().enablePreview()) {
```

Comment:
if you add enable-preview support you'll need to enable on the compile side too. otherwise wont do much.